### ISSUE TITLE
meta.Version is always 1.0.0 on android

### ISSUE THREAD

**Issue by bon-ami** [fyne-io/fyne#3109]
### Describe the bug:

I fail to get appVersion on Android by code. But the same code works on Windows.
The version in detail info of my app, such as in Settings, shows the version correctly.

### To Reproduce:
`fyne package -os android -appVersion 1.1.1` with FyneApp.toml setting other info, such as Build.

### Screenshots:

Refer to my [screenshot](https://gitlab.com/bon-ami/te/-/blob/main/verand.png) for [my sample project](https://gitlab.com/bon-ami/te)

### Example code:

```go
	myApp := app.NewWithID("a")
	myWin := myApp.NewWindow("a")
	meta := myApp.Metadata()  // it's always 1.0.0
	but1 := widget.NewButton("StrFlw", nil)
	but2 := widget.NewButton("StrFlw", nil)
	tabs := container.NewAppTabs(
		container.NewTabItem(meta.Version, container.NewVBox(but1)),  // show it as tab name
		container.NewTabItem("2", container.NewVBox(but2)),
		//container.NewTabItem("3", makeControlsCfg(myWin)),
	)
	myWin.ShowAndRun(tabs)
//	myApp.Run()
```

### Device (please complete the following information):
 - **OS:** Android app built on Windows 10
 - **Version:** Android 9
 - **Go version:** 1.18
 - **Fyne version:** 2.2.1

**Comment by andydotxyz** [comment]
Can you try v2.2.2 please? Does the version become always 0.0.1?

**Comment by bon-ami** [comment]
@andydotxyz , yes. Fyne 2.2.2 makes the version 0.0.1 regardless of what I set it to.

**Comment by andydotxyz** [comment]
That PR should have the fix in if you'd like to test.
Install the fyne executable from that code should fix the metadata injection for mobile builds.

**Comment by andydotxyz** [comment]
This is landed and will be in `v2.2.3`

**Comment by bon-ami** [comment]
confirmed to be solved on 2.2.3. Thanks.
